pkg/apis/crd.projectcalico.org/v1/ippool.go

```diff
@@ -103,11 +104,9 @@ const (
 	IPIPModeCrossSubnet IPIPMode = "CrossSubnet"
 )
 
-type AssignmentMode string
-
 const (
-	Automatic AssignmentMode = "Automatic"
-	Manual    AssignmentMode = "Manual"
+	Automatic operatorapi.AssignmentMode = "Automatic"
```

Comment:
These should be defined in the operator API package rather than here.

Reply:
We should rename both of these constants - the convention for enums is that they should include the type in the name to avoid this type of conflict.

```go
type AssignmentMode string

const (
  AssignmentModeAutomatic = "Automatic"
  AssignmentModeManual = "Manual"
)
```